File: lat2_java/product.java
class product {
    private String price;
    private String idProduct;
    
    public product(){
    }

    public product(String price, String idProduct){
        this.price = price;
        this.idProduct = idProduct;
    }

    public String getPrice(){
        return price;
    }

    public void setPrice(String price){
        this.price = price;
    }

    public String getIdProduct(){
        return idProduct;
    }

    public void setIdProduct(String idProduct){
        this.idProduct = idProduct;
    }

    public void printProduct(){
        System.out.println("");
        System.out.print("Price : ");
        System.out.println(getPrice());
        System.out.print("Id Product : ");
        System.out.println(getIdProduct());
    }
    

}

File: lat2_PHP/product.php
<?php

class product{

	private $price = "";
	private $idProduct = "";

	public function __construct($price = "", $idProduct = ""){
		$this->price = $price;
		$this->idProduct = $idProduct;
	}

	public function setprice($price){
		$this->price = $price;
	}

	public function getprice(){
		return $this->price;
	}

	public function setidProduct($idProduct){
		$this->idProduct = $idProduct;
	}

	public function getidProduct(){
		return $this->idProduct;
	}

	public function printProduct(){
		echo "<br>";
		echo "Price 		: ". getprice() ."<br>";
		echo "idProduct 		: ". getidProduct() ."<br>";
	}
}